docs/sources/clients/k6/query-scenario.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Query testing
+weight: 30
+---
+# Query testing
+
+When designing a test scenario for load testing the read path of a Loki
+installation, it is important to know what types of queries you expect.
+
+## Supported query types
+
+Loki has 5 different types of queries that can be performed:
+
+* instant query
+* range query
+* labels query
+* label values query
+* series query
+
+In a real-world use-case, such as querying Loki using it as a Grafana
+datasource, all of these queries are used. Each of them has a different
+[API]({{< relref "../../api/_index.md" >}}) endpoint. The xk6-loki extension
+provides a [Javascript API](https://github.com/grafana/xk6-loki#javascript-api)
+for all these query types.
+
+> **Note:** The query strings, labels and matchers in the following code fragments need to be adjusted to your own data.
```

Comment:
I don't think this note is needed at all. All developers will understand that these example code fragments have queries in them which must be substituted for their own.

In addition, the phrase "in the following code fragments" is out of place (style-wise). Forward references should not be to items in other sections.   

If you feel you must keep this, then do not display it as a "Note," and replace "in the following code fragments" with "in the example code fragments" .

```suggestion
```